### PR TITLE
rfc filter sliders

### PR DESCRIPTION
**Pull request by haslinghuis**
Depends on:
https://github.com/betaflight/betaflight/pull/11038 (merged)
https://github.com/betaflight/betaflight/pull/11050

Fixes: #2611 
Fixes: #2620
<details>
  <summary>changes</summary>
Firmware Sliders are setting Gyro and DTerm filters now.

Observed behaviour with current filters:

1 Now changing lowpass 1 or static lowpass or lowpass 2 dynamic filter type disables gyro slider.
2 Now changing a lowpass 1, static or lowpass 2 values disables gyro / dterm slider.

This PR changes behavior to:

- [x] Decoupling gyro and dterm sliders in GUI to retain separate control (@mikeller).
- [x] Use firmware sliders for any valid filter combination (@ctzsnooze / @sugaarK).
- [x] Trying to get right values from firmware after switch change
- [x] Removed the need to save after switching filters.
- [x] Fixed enabling gyro slider with a hack.
- [x] Fixed enabling dterm slider with a hack.
- [x] Fix saving lowpass 1 type.
- [x] Reworked to fix html for legacy and firmware
- [x] Bring back some old i18n

Okay - we decided to change the whole thing (@ctzsnooze thanks for functional design)
</details
Screenshot from 2021-11-11 06-53-21

Builds:
https://dev.azure.com/Betaflight/Betaflight%20Nightlies/_build/results?buildId=4419&view=artifacts&pathAsName=false&type=publishedArtifacts

### CODE REVIEW

**Comment by haslinghuis** [comment]
Wow Sonar is really getting picky now.

**Comment by haslinghuis** [comment]
Needs some more work but basically on primary values it works

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by spatzengr** [comment]
@haslinghuis / @ctzsnooze ,

Do we get the same behavior when using the OSD sliders?

**Comment by haslinghuis** [comment]
Will push more today

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by limonspb — DISMISSED** [review]

**Review by ctzsnooze — DISMISSED** [review]

**Comment by ctzsnooze** [comment]
Fantastic work.  End result is really intuitive and simple to use.  Well worth the effort!  Tested and works great.

**Comment by haslinghuis** [comment]
Sorry merge conflicts.

**Review by blckmn — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Pull request merged**
